customcontext/context.cpp

```diff
@@ -302,15 +262,25 @@ void Context::renderContextInitialized(QSGRenderContext *ctx)
 
 #endif // CUSTOMCONTEXT_HYBRISTEXTURE
 
-    QSGContext::renderContextInitialized(ctx);
+    CONTEXT_CLASS_BASE::renderContextInitialized(ctx);
 }
 
 
 
 
 
-void CONTEXT_CLASS::initialize(QOpenGLContext *context)
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+void RenderContext::initialize(const InitParams *inCtx)
 {
+#elif QT_VERSION >= 0x050800
+void RenderContext::initialize(void *inCtx)
+{
+    QOpenGLContext *context = static_cast<QOpenGLContext*>(inCtx);
+#else
+void RenderContext::initialize(QOpenGLContext *inCtx)
```

Comment:
Unfortunately still not compiling:
```
context.cpp:280:6: error: no declaration matches ‘void CustomContext::RenderContext::initialize(QOpenGLContext*)’
 void RenderContext::initialize(QOpenGLContext *inCtx)
      ^~~~~~~~~~~~~
In file included from context.cpp:42:
context.h:93:10: note: candidate is: ‘void CustomContext::RenderContext::initialize(void*)’
     void initialize(void *context);
          ^~~~~~~~~~
context.h:82:7: note: ‘class CustomContext::RenderContext’ defined here
 class RenderContext : public RENDER_CONTEXT_CLASS_BASE
       ^~~~~~~~~~~~~
```

But now it's easier to see the problem. The method definition in the header seem reversed with >=5.8 and older vs the actual use here.